fe/src/pages/profile.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <div
+    class="py-6 px-36 md:px-44 min-h-screen lg:px-48 bg-linear-gradient-white-to-light overflow-auto"
+  >
+    <base-user-card
+      :block-chain-value="blockChainValue"
+      :block-chain-label="blockChainLabel"
+      :img-src="imgSrc"
+      :user-id="userId"
+    />
+    <div class="mt-6">
+      <base-tabs
+        :active-tab="activeTab"
+        :items="profileTabs"
+        @change-tab="activeTab = $event"
+      />
+    </div>
+    <div v-if="activeTab === 'My Campaigns'">
+      <div
+        class="mt-6 grid grid-cols-3 gap-6 sm:grid-cols-1 lg:grid-cols-2 xl:grid-cols-3 campaigns"
+      >
+        <base-campaign-card
+          v-for="i in 6"
+          :key="i"
+          :title="title"
+          :description="description"
+          :eth-value="ethValue"
+          :img-src="imgSrc"
+          :days-left="daysLeft"
+        ></base-campaign-card>
+      </div>
+      <div class="w-full mt-8 h-14 flex items-center justify-center">
+        <BasePaginator
+          :current-page="currentPage"
+          :last-page="10"
+          :per-page="first"
+          :on-page-change="setPage"
+        />
+      </div>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import UserCardProps from "~/types/UserCardProps";
+import UserCardSample from "~/mocks/user-card-sample.json";
+import CampaignCardProps from "~/types/CampaignCardProps";
+import CardSample from "~/mocks/card-sample.json";
+
+const userCardValueSample = ref<UserCardProps>(UserCardSample);
+
+const { userId, imgSrc, blockChainLabel, blockChainValue } =
+  userCardValueSample.value;
+
+const first = ref(5);
```

Comment:
Name is too vague and not clear as to what this is.
Please update the name of this constant to something more descriptive.
Example only: 

```suggestion
const firstPage = ref(5);
```